README.md

```diff
@@ -22,12 +22,12 @@ This command sets the environment for your local copy of the application to deve
 export ASPNETCORE_ENVIRONMENT="Development"
 ```
 
-On initial installation of the Banagazon web application you must set an environment variable to your local database. This variable must point the application to the database
+On initial installation of the Banagazon web application you must set an environment variable to your local database. Even though you have not yet created a database, this variable will set the location of your database for later after creation.
```

Comment:
Wording seems a little weird. Maybe something like:

On initial installation of the Banagazon web application you must set an environment variable to your local database.  _(This database with the file name `bangazon.db` will be created when you run `dotnet ef database update` later in the installation process.)_